course_1_1/Program.cs

```diff
@@ -10,19 +10,594 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Please enter a radius");
-            string rstr = Console.ReadLine();
-            int.TryParse(rstr, out int r);
-            Console.WriteLine($"The square of the round {Program.PerformCalculation(r)}");
-            
+
+            //
+            //WorkWithNumber();
+            //
+            //
+            //DelNumeral();
+            //
+            //
+            //EvenNumbers();
+            //
+            //
+            //Average();
+            //
+            //
+            //RandomNumber();
+            //
+            //
+            //Exponentiation();
+            //
+            //
+            //ProductRoute();
+            //
+            //
+            //DrowFigure();
+            //
+            //
+            //FibonachiNumberLessThan();
+            //
+            //
+            //FirstNumbersFibonachi();
+            //
+            //////////DrawPyramid();
+            //
+            //
+            //SumRow();
+            //
+            //BetweenNumbers();
+            //
+            //SquareLessThanNumber();
+            //
+            //
+            //SyracuseTheorem();
+            //
+            //
+            //MileageSkier();
+            //
+            //
+            //ReverseNumber();
+            //
+            //
+            //NumeralsOfNumber();
+            //
+            //
+            //HighestDigitOfNnumber();
+            //
+            //
+            //UnsignedMultiplication();
+            //
+            //
+            DrowFigure();
+            //
+            //
+            //SumRow();
+        } 
```

Comment:
As you know, a code is written for people.
And in this case for me. It is very difficult to read and understand the logic when code is continuous text.
Code should be aligned, should have empty lines when you want to emphasize something.
Please try to use such approach.